UE4SS/include/LuaType/LuaCustomProperty.hpp

```diff
@@ -34,6 +35,10 @@ namespace RC::LuaType
           public:
             auto add(StringType property_name, std::unique_ptr<Unreal::CustomProperty>) -> void;
             auto clear() -> void;
+
+            using ForEachCallable = std::function<bool(LuaCustomProperty const&)>;
+            auto for_each(Unreal::UObject* base, ForEachCallable callable) -> bool;
```

Comment:
```suggestion
            auto for_each(Unreal::UObject* base, const ForEachCallable& callable) -> bool;
```
Take std::function as const reference to avoid copy.